coverage_point_calculator/src/location.rs

```diff
@@ -31,17 +31,11 @@ pub fn asserted_distance_to_trust_multiplier(
             76..=100 => dec!(0.25),
             _ => dec!(0.00),
         },
-        RadioType::IndoorCbrs => dec!(1.0),
-        RadioType::OutdoorCbrs => dec!(1.0),
     }
 }
 
-pub(crate) fn average_distance(radio_type: RadioType, trust_scores: &[LocationTrust]) -> Decimal {
-    // CBRS radios are always trusted because they have internal GPS
-    if radio_type.is_cbrs() {
-        return dec!(0);
-    }
-
+pub(crate) fn average_distance(trust_scores: &[LocationTrust]) -> Decimal {
+    // FIXME-K: if count = 0, division by zero happens
```

Comment:
I have a vague memory of @bbalser telling me that the `valid_radios.sql` query will not return radios that have empty location trust arrays. 

But it was a long time ago, and I could be remembering something totally different.

That being said, when this is fixed, I'm not sure what a good value to return would be, as `0` is a best case scenario.

Reply:
given the query is from a single table, and the values in question are not nullable, it is not possible to have rows return that from the table and have an empty array.

Reply:
> That being said, when this is fixed, I'm not sure what a good value to return would be, as 0 is a best case scenario.

If array is empty it doesn't return 0, it panics.

I think the best way is just make returning Result<Decimal> and return error if empty array.
It will be better than **panic** in code